test.js

```diff
@@ -49,3 +49,7 @@ test('supports unicode surrogate pairs', t => {
 test('doesn\'t add unnecessary escape codes', t => {
 	t.is(m('\u001B[31municorn\u001B[39m', 0, 3), '\u001B[31muni\u001B[39m');
 });
+
+test('support true color escape sequences', t => {
+	t.is(m('\u001B[[1m\u001B[[48;2;255;255;255m\u001B[[38;2;255;0;0municorn\u001B[[39m\u001B[[49m\u001B[[22m', 0, 3), '\u001B[1m\u001B[48;2;255;255;25m\u001B[38;2;255;0;0muni\u001B[39m');
```

Comment:
```suggestion
	t.is(sliceAnsi('\u001B[[1m\u001B[[48;2;255;255;255m\u001B[[38;2;255;0;0municorn\u001B[[39m\u001B[[49m\u001B[[22m', 0, 3), '\u001B[1m\u001B[48;2;255;255;25m\u001B[38;2;255;0;0muni\u001B[39m');
```

Appears the imported module changed name on master, I think renaming it should fix the CI

Reply:
> Appears the imported module changed name on master, I think renaming it should fix the CI

In fact, it does! Tests are passing again! :-D